support-cycle.md

```diff
@@ -0,0 +1,180 @@
+
+ - We will support IE11 until around 2025-10-14
+
+#### Opera, Maxthon, Camino, Konqueror, OmniWeb, <Insert Browser Here>
+
+We do not support any of these browsers officially, this is because their usage is too small or because they use the same JS/HTML engines as already supported engines.
+
+#### Rhino JS, Node ChakraCore, <Insert Server Side Engine Here>
+
+We only officially support Node.js. The usage share of alternative engines is too small for us to consider supporting them.
+
+### Questions
+
+#### What if I want support of an older version of a project?
+
+If you are using an older version of one of the Chai projects - for example Chai@3.5.0 - and wish for us to backport fixes, this can be negotiated with the team for that project. The team responsible for the maintainance of the project will provide a cost breakdown of the engineering time it would require to backport any bugfixes to older versions and make a release which would need to be paid upfront for the work to commence. For example if you wanted a bugfix from chai@4.1.1 backported to chai@3.5.0, we would estimate the number of days it would take 1 maintainer to make this fix, and multiply it by the going market day rate, in US$. You should fully expect this cost to be in thousands-of-dollars cost region. Please note any work paid for to backport features will be made open source, and as such all users will benefit from this.
+
+All of our projects are open source, and so if you disagree with the pricing we quote you, you are of course welcome to solicit other developers to make the neccessary changes and raise a pull request.
+
+#### What if I want a project to support an older platform?
+
+If you would like the latest version of a project to support an older platform, for example Chai@4 to support IE9 - and you wish for us to maintain this, then this can be negotiated with the team for that project. The team responsible for the maintainance of the project will provide a cost breakdown of the engineering time it would require to change the project's code to support the older platform which would need to be paid upfront for the work to commence. Additional on-going costs will incur for continuing to support the older platform for the duration of the major version of that project. For example if you wanted a chai@4 to support IE9, we would estimate the number of days it would take 1 maintainer to make the necessary code changes and multiply by the going market day rate, in US$, as well as estimate a monthly retainer (some amount of days multiplied by the market day rate in US$) in order to continue support until Chai@5 was released. The on-going cost would need to be made regularly to ensure we are able to continue to support the older platform. You should fully expect this cost to be in tens-of-thousands-of-dollars cost region. Please note any work paid for to support additional platforms will be made open source, and as such all users will benefit from this.
```

Comment:
Just noting that personally I probably won't be able to do something like this, even if money is on the table.

Reply:
Again, this was just here to trigger the discussion. I felt the need to document this as a way to rebuff support requests that are outside of our remit. 

Is this something we feel comfortable drawing a line in the sand about? I understand our various _real_ jobs might prevent some of us from committing to doing external paid work, or even just having the flexibility for time off. I certainly don't want to promote a position for those who have the privilege of flexibility/agency to benefit over those that don't, but I am also very much keen on seeing the maintainership of Chai being rewarded more concretely (fiscally) for your hard work. 

I would definitely like to hear more people's thoughts on all of this. Perhaps this section shouldn't be included in this document? Perhaps we should take a harder line and simple document that none of this work should be undertaken?